cirq/experiments/xeb_simulation.py

```diff
@@ -46,7 +45,7 @@ class _Simulate2qXEBTask:
 class _Simulate_2q_XEB_Circuit:
     """Closure used in `simulate_2q_xeb_circuits` so it works with multiprocessing."""
 
-    def __init__(self, simulator: 'cirq.SimulatesIntermediateState'):
+    def __init__(self, simulator: 'sim.SimulatesIntermediateState'):
```

Comment:
Should be able to switch back to `cirq.{...}` here and below.

Reply:
Done

Reply:
How to I test QSimSimulator?

Reply:
Make sure you have both the latest version of Cirq (including this change) and `qsimcirq`, then run any `SimulatesFinalState` test with `QSimSimulator` in place of the Cirq simulator. See the [qsimcirq tests](https://github.com/quantumlib/qsim/blob/master/qsimcirq_tests/qsimcirq_test.py) for examples.

Reply:
I'm befuddled. Installed qsim from source. Tried changing `test_intermediate_simulator` to use `QSimSimulator`, but it can't find `QSimSimulator`. How do I get the test to find `QSimSimulator`? Is this even what you mean?

Reply:
Nvm, got it (I think): Added the following to beginning of `qsimcirq_test` in the qsim project:
```
import sys
sys.path.insert(1, '/home/dax/cirq')
```
Then ran those unit tests, and they passed. I updated my cirq code to throw an error just to verify it was being called, and the error was thrown, so confirms it was actually calling the new code.

Conclusion: the new class type declarations work fine with the external simulators.